examples/withsequence-recursion.yaml

```diff
@@ -0,0 +1,43 @@
+# This example shows how to use recursion in a Workflow.
```

Comment:
```suggestion
# This example shows how to combine recursion and a withSequence loop in a Workflow.
```

since you modified the name now, we should probably modify the comment a little